shared/middleware/channels_api.rb

```diff
@@ -199,6 +199,40 @@ class ChannelsApi < Sinatra::Base
     {publishedAt: nil}.to_json
   end
 
+  #
+  # POST /v3/channels/<channel-id>/disable_content_moderation
+  #
+  # Disables automatic content moderation.
+  #
+  post %r{/v3/channels/([^/]+)/disable-content-moderation} do |channel_id|
+    not_authorized unless project_validator?
+    dont_cache
+    content_type :json
+    begin
+      value = StorageApps.new(storage_id('user')).set_content_moderation(channel_id, true)
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      bad_request
+    end
+    {skip_content_moderation: value}.to_json
+  end
+
+  #
+  # POST /v3/channels/<channel-id>/enable_content_moderation
+  #
+  # Enables automatic content moderation.
+  #
+  post %r{/v3/channels/([^/]+)/enable-content-moderation} do |channel_id|
+    not_authorized unless project_validator?
+    dont_cache
+    content_type :json
+    begin
+      value = StorageApps.new(storage_id('user')).set_content_moderation(channel_id, false)
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      bad_request
+    end
+    {skip_content_moderation: value}.to_json
+  end
```

Comment:
These two blocks are identical except for the route name and the boolean value being set.  Let's extract a helper method!